boards/BMS/BMS.ioc

```diff
@@ -1,208 +1,108 @@
+FREERTOS.IPParameters=Tasks01,MEMORY_ALLOCATION,FootprintOK,configUSE_TRACE_FACILITY,INCLUDE_vTaskDelayUntil
 FREERTOS.MEMORY_ALLOCATION=1
-FREERTOS.Tasks01=Task1Hz,-2,TASK1HZ_STACK_SIZE,RunTask1Hz,Default,NULL,Static,Task1HzBuffer,Task1HzControlBlock;Task1kHz,1,TASK1KHZ_STACK_SIZE,RunTask1kHz,Default,NULL,Static,Task1kHzBuffer,Task1kHzControlBlock;TaskCanRx,-3,TASKCANRX_STACK_SIZE,RunTaskCanRx,Default,NULL,Static,TaskCanRxBuffer,TaskCanRxControlBlock;TaskCanTx,-3,TASKCANTX_STACK_SIZE,RunTaskCanTx,Default,NULL,Static,TaskCanTxBuffer,TaskCanTxControlBlock;Task100Hz,-1,TASK100HZ_STACK_SIZE,RunTask100Hz,Default,NULL,Static,Task100HzBuffer,Task100HzControlBlock
-FREERTOS.configCHECK_FOR_STACK_OVERFLOW=2
-FREERTOS.configENABLE_BACKWARD_COMPATIBILITY=1
-FREERTOS.configGENERATE_RUN_TIME_STATS=0
-FREERTOS.configIDLE_SHOULD_YIELD=1
-FREERTOS.configLIBRARY_LOWEST_INTERRUPT_PRIORITY=15
-FREERTOS.configLIBRARY_MAX_SYSCALL_INTERRUPT_PRIORITY=5
-FREERTOS.configMAX_CO_ROUTINE_PRIORITIES=2
-FREERTOS.configMAX_PRIORITIES=7
-FREERTOS.configMAX_TASK_NAME_LEN=16
-FREERTOS.configMINIMAL_STACK_SIZE=128
-FREERTOS.configQUEUE_REGISTRY_SIZE=8
-FREERTOS.configTICK_RATE_HZ=1000
-FREERTOS.configUSE_APPLICATION_TASK_TAG=0
-FREERTOS.configUSE_COUNTING_SEMAPHORES=0
-FREERTOS.configUSE_CO_ROUTINES=0
-FREERTOS.configUSE_DAEMON_TASK_STARTUP_HOOK=0
-FREERTOS.configUSE_IDLE_HOOK=0
-FREERTOS.configUSE_MALLOC_FAILED_HOOK=0
-FREERTOS.configUSE_MUTEXES=1
-FREERTOS.configUSE_PREEMPTION=1
-FREERTOS.configUSE_RECURSIVE_MUTEXES=0
-FREERTOS.configUSE_STATS_FORMATTING_FUNCTIONS=0
-FREERTOS.configUSE_TASK_NOTIFICATIONS=1
-FREERTOS.configUSE_TICKLESS_IDLE=0
-FREERTOS.configUSE_TICK_HOOK=1
-FREERTOS.configUSE_TIMERS=0
+FREERTOS.Tasks01=Task100Hz,-1,TASK100HZ_STACK_SIZE,RunTask100Hz,Default,NULL,Static,Task100HzBuffer,Task100HzControlBlock;TaskCanRx,-3,TASKCANRX_STACK_SIZE,RunTaskCanRx,Default,NULL,Static,TaskCanRxBuffer,TaskCanRxControlBlock;TaskCanTx,-3,TASKCANTX_STACK_SIZE,RunTaskCanTx,Default,NULL,Static,TaskCanTxBuffer,TaskCanTxControlBlock;Task1kHz,0,TASK1KHZ_STACK_SIZE,RunTask1kHz,Default,NULL,Static,Task1kHzBuffer,Task1kHzControlBlock;Task1Hz,-2,TASK1HZ_STACK_SIZE,RunTask1Hz,Default,NULL,Static,Task1HzBuffer,Task1HzControlBlock
```

Comment:
The user constants for stack size need to be input into the .ioc manually, trying to add them in the GUI does not work